ts/src/pro/bingx.ts

```diff
@@ -224,6 +237,207 @@ export default class bingx extends bingxRest {
         }, market);
     }
 
+    async watchTickers (symbols: Strings = undefined, params = {}): Promise<Tickers> {
+        /**
+         * @method
+         * @name bingx#watchTickers
+         * @description watches a price ticker, a statistical calculation with the information calculated over the past 24 hours for all markets of a specific list
+         * @see https://bingx-api.github.io/docs/#/en-us/swapV2/socket/market.html#Subscribe%20to%2024-hour%20price%20changes%20of%20all%20trading%20pairs
+         * @param {string[]} symbols unified symbol of the market to watch the tickers for
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @returns {object} a [ticker structure]{@link https://docs.ccxt.com/#/?id=ticker-structure}
+         */
+        await this.loadMarkets ();
+        symbols = this.marketSymbols (symbols, undefined, true, true, false);
+        let firstMarket = undefined;
+        let marketType = undefined;
+        const symbolsDefined = (symbols !== undefined);
+        if (symbolsDefined) {
+            firstMarket = this.market (symbols[0]);
+        }
+        [ marketType, params ] = this.handleMarketTypeAndParams ('watchTickers', firstMarket, params);
+        if (marketType === 'spot') {
+            throw new NotSupported (this.id + ' watchTickers is not supported for spot markets yet');
+        }
+        const messageHashes = [];
+        const subscriptionHashes = [ 'all@ticker' ];
+        if (symbolsDefined) {
+            for (let i = 0; i < symbols.length; i++) {
+                const symbol = symbols[i];
+                const market = this.market (symbol);
+                messageHashes.push (this.getMessageHash ('ticker', market['symbol']));
+            }
+        } else {
+            messageHashes.push (this.getMessageHash ('ticker'));
+        }
+        const url = this.safeString (this.urls['api']['ws'], marketType);
+        const uuid = this.uuid ();
+        const request: Dict = {
+            'id': uuid,
+            'dataType': 'all@ticker',
+        };
+        if (marketType === 'swap') {
+            request['reqType'] = 'sub';
+        }
+        const result = await this.watchMultiple (url, messageHashes, this.deepExtend (request, params), subscriptionHashes);
+        // for efficiency, we have two type of returned structure here - if symbols array was provided, then individual
```

Comment:
I don't understand this comment, inside handleTicker we always resolve individual tickers so this does not make sense